src/extensions/averagewaveforms/AverageWaveformsView/setupElectrodes.ts

```diff
@@ -112,20 +127,21 @@ const setupElectrodes = (args: {width: number, height: number, electrodeLocation
     if (layoutMode === 'vertical') {
         return setupVerticalElectrodes({width, height, electrodeIds})
     }
+    const electrodeLocations2 = fixDegenerateCase(electrodeLocations)
```

Comment:
Might be more descriptive to call this something like 'correctedElectrodeLocations', or just have it be `let realizedElectrodeLocations` (l. 139) from the start

Reply:
you're right, I'll update that